internal/femto/server_test.go

```diff
@@ -119,14 +119,15 @@ func TestGetHappyPath(t *testing.T) {
 	assert.JSONEq(t, `{"X": 101}`, string(j))
 }
 
-type unit struct{}
+type unit = struct{}
+type other_unit = struct{}
```

Comment:
NIT: Go nameing says this should be `otherUnit`, but also it should just be deleted